src/data-types.js

```diff
@@ -830,10 +830,16 @@ class GEOMETRY extends ABSTRACT {
     this.srid = options.srid;
   }
   _stringify(value, options) {
-    return `GeomFromText(${options.escape(wkx.Geometry.parseGeoJSON(value).toWkt())})`;
+    if (this.srid) {
+      return `ST_GeomFromText(${options.escape(wkx.Geometry.parseGeoJSON(value).toWkt())}, ${this.srid})`;
```

Comment:
From MySQL 5.7 https://dev.mysql.com/doc/refman/5.7/en/gis-wkt-functions.html

It seems `ST_GeomFromText` is synonyms for `GeomFromText`, I remember some time ago this was not true. Earlier in the main issue I asked for checking dialect version here, but that is no longer required.

Reply:
Yep, and as I also mentioned in the commit corresponding to this change:

> MariaDB added support for ST_GeomFromText in 5.3.3 according to https://mariadb.com/kb/en/mysqlmariadb-spatial-support-matrix/